ext/pgsql/tests/bugGH12763.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Bug #GH12763 (pg_untrace(): Argument #1 ($connection) must be of type resource or null, PgSql\Connection given)
```

Comment:
Nit: for GitHub bugs, use the `gh12763.php` as filename. Similarly the title should be `GH-12763 (pg_untrace(): ...)`
For bugs on the old bugtracker we use `Bug #xxxxx` title and `bugxxxxx.phpt` filename.